### ISSUE TITLE
snel inzicht in resultaat

### ISSUE THREAD

**Issue by PalmJanssen** [Geonovum/NEN3610-Linkeddata#83]
Door Remco in 't Hout

## Waar
Algemeen / niet specifiek

## Samenvatting
Visueel overzicht van concepten in NEN3610, UML naar Linked data

## Beschrijving

Het is   in het document moeilijk snel inzicht te krijgen in het resultaat. Per   onderwerp/concept worden zaken uitgebreid weergegeven, maar overzicht over   het geheel zie ik niet staan (of kijk ik ergens overheen?). Zo'n overzicht   zou helpen om snel het resultaat te begrijpen en te  beoordelen.

## Oplossingsrichting

Visualisatie   van de huidige NEN 3610 concepten naast de vertaling naar linked data   ontologie/informatiemodel. Eventueel opgesplitst in een aantal delen als het   geheel te complex is om weer te geven.

**Comment by PalmJanssen** [comment]
De transformatieregels worden beschreven vanuit de source informatie-elementen  uit NEN 3610. dit gebeurt in paragraaf 7.3.4 en verder: https://geonovum.github.io/NEN3610-Linkeddata/#standaardregels.
Elke NEN 3610 informatie-element is een aparte paragraaf. Hiermee hebben we geprobeerd om het overzichtelijk en snel vindbaar te maken.

**In hoofdstuk 8 worden de drie LD vertalingen van het IMGOLF behandeld. het source model NEN 3610 UML IMGOLF is hier niet te zien. Mogelijk beter om dit toe te voegen.**

**Comment by PalmJanssen** [comment]
De volgende veranderingen zijn verwerkt:
- inleiding aangepast voor beter overzicht
- UML IMGOLF in hfst 8 toegevoegd.
- tabel met UML - RDF vertaling op hoofdlijn wordt toegevoegd

**Comment by PalmJanssen** [comment]
tabel met UML - RDF vertaling op hoofdlijn wordt toegevoegd
@architolk is dat te doen een tabel toevoegen met de source target transformatie op hoofdlijn?

**Comment by architolk** [comment]
Ja, ik denk het wel. Zie bv bij MIM een vergelijkbare tabel: https://geonovum.github.io/MIM-Werkomgeving/#overzicht
